src/main/java/frc/robot/LogitechGamingPad.java

```diff
@@ -26,230 +26,207 @@
 	/**
 	 * The Joystick class is utilized for simplicity.
 	 */
-	private Joystick gamepad;
-
-	/**
-	 * Construct an instance of a Logitch Gaming Pad. The Logitech Gaming Pad index
+  private Joystick gamepad;
+
+	// Constants
```

Comment:
nit: deindent this line, and all comment blocks basically. this prob won't pass CI?

Reply:
actually it did - for code style the comments should be at the same indent level as the code. this may need to be added to checkstyle